lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetSchemaConverter.java

```diff
@@ -66,14 +66,18 @@
         }
     }
 
+    public static final boolean HIVE_PARQUET_USE_LEGACY_DECIMAL_ENCODING = true;
```

Comment:
nits in cmt msg "Use binary encoding for decimals in hive parquet optimized writer"

- binary encoding -> byte encoding ??
- DeltaLake -> Delta Lake
- interger -> integer
- makes fixes -> fixes
- in encoded -> encoded